store/views.py

```diff
@@ -77,6 +105,60 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    response_data = {
+        'message': None,
+    }
+   
+    book_id = request.POST.get('bid')
+
+    try:
+        book_copy = BookCopy.objects.get(id=book_id)
+        response_data['message'] = 'success'
+        book_copy.borrower = None
+        book_copy.borrow_date = None
+        book_copy.status = True
+        book_copy.save()
+    except:
+        response_data['message'] = 'failure'
+        
+
+    return JsonResponse(response_data)
+
+
+@csrf_exempt
+@login_required
+def rateBookView(request):
+    response_data = {
+        'message': None,
+    }
+
+    if request.POST:
+        book_id = request.POST.get('bid')
+        book_rate = request.POST.get('brate')
+
+        books = Book.objects.get(id=book_id)
+        rating1 = BookRating.objects.filter(book=books, user=request.user)
+
+        if len(rating1)==0:
+            new_rating = BookRating(book=books, user=request.user, rating=book_rate)
```

Comment:
While this is correct `get_or_create` could be used here for better code clarity.